src/_forms.scss

```diff
@@ -0,0 +1,259 @@
+/**
+ * Outer wrapper class for all input fieldsets
+ */
+.field {
+    margin: 1.5em 0;
+}
+
+/**
+ * Wrapper class for all input fields
+ */
+.input, 
+.textarea, 
+.select {
+    margin: 0;
+    position: relative;
+    overflow: hidden;
+
+    /**
+    * Underline for input fields
+    */
+    &::after, 
+    &::before {
+        content: '';
+        width: 100%;
+        border-bottom: 2px solid #222;
+        position: absolute;
+        bottom: 0;
+        left: 0;
+    }
+
+    /**
+    * After pseudo element for input underlines
+    * initially hidden outside by parent class
+    * overflow: hidden
+    * 1. Move the pseudo element to the left by 100%
+    * 2. Add animation effects to transform style
+    */
+    &::after {
+        margin-top: 1px;
+        border-bottom: 4px solid #51AFC1;
+        -webkit-transform: translate3d(-101%, 0, 0); /* 1 */
```

Comment:
Can you check that this and any other vendor prefixed properties are not added in by the gulpfile? If they are we can remove them, which will make this more compact as well